### PR TITLE
Create CCC_TAC_Minutes-2022-05-05.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+* Mingliang Pei (Ming) (Broadcom)
+* Hannes Tschofenig (Arm)
+* Mark Novak (JP Morgan Chase)
+* Naveen Cherukuri (NVIDIA)
+* Yogesh Deshpande (Arm)
+* Nick Vidal (Profian)
+* Simon Johnson (Intel)
+
+#### Also Participating  
+ * None
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md:66]: Not sure what this section means
```suggestion
```

File: TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+### TAC Tech Talks
+ *  IETF Trusted Execution Environment Provisioning (TEEP)
+ *  Presented by - Mingliang Pei (MP), Hannes Tschofenig (HT), Dave Thaler (DT)
+
+
+### Open Action Items
+1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has opened a PR for review https://github.com/confidential-computing/governance/pull/92
+
+1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md:87]:
```suggestion
1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

File: TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+* Dan Middleton (DM) - many new interns introduced in slack
+* Nick Vidal (NV) - many people interested in Confidential Computing, Veracruz and Enarx participating in Outreachy, Veracruz selected two candidates, Enarx selected one, internships last for 3 months, Enarx interns have published articles that are getting a lot of attention, next round, maybe have interns go to the project slack channels so they will be better oriented, details in the Outreach newsletter 
+
+### Common Test Infrastructure
+ * DT - existing $15K budget for a GitHub CI/CD actually running on hardware, multiple projects could share that cost, need person to maintain that hardware, proposing LF person to maintain, asked Brian Warner (LF) and Stephan Walli (Governing Board), wondering if it would need a full person, asked for thoughts for moving this forward, proposing raising this amount
+ * DM - might be good to get the maintainers involved
+ 
+ ***ACTION*** mentors to reach out to the project for getting the maintainers involved
+ 
+ * NV - Enarx has benefited form hiring a person, looking at hiring an intern, supports a LF person managing GitHub CI/CD
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md:115]:
```suggestion
 * NV - Enarx has benefited from hiring a person, looking at hiring an intern, supports an LF person managing GitHub CI/CD
```

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md (hunk 1 above)
Comment (krtaylor) [inline TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md]:
```suggestion

```

File: TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+**Proposed:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf) and [April 21, 2022](../2022-04-21/TAC_Minutes-2022-04-21.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+ * Quorum was not reached and the minutes were not approved.
+ 
+#### Pending approval
+ * None
+
+### TAC Tech Talks
+ *  IETF Trusted Execution Environment Provisioning (TEEP)
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-05-05/CCC_TAC_Minutes-2022-05-05.md:77]:
```suggestion
 *  [IETF Trusted Execution Environment Provisioning (TEEP)] (TeepOverview%20v5.pptx)
```

**Pull request merged**